spec/index.bs

```diff
@@ -7313,6 +7315,8 @@ dictionary GPURenderPassDepthStencilAttachment {
          a single [=mipmap level=], single [=array layer=], and all the [=aspects=] in the texture.
     - |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/usage}}
         must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
+    - If |this|.{{GPURenderPassDepthStencilAttachment/depthLoadValue}} is a `float`, it must be
```

Comment:
If it is an integer shouldn't it still need to be between 0 and 1 inclusive?

Reply:
It can't be an integer, it can only be `(GPULoadOp or float)`

Reply:
Similarly in the definition:
> If a float, indicates the value to clear view's depth component to prior to executing the render pass.

Both could be clearer though.